src/query/ast/src/ast/statements/copy.rs

```diff
@@ -305,7 +305,11 @@ impl Connection {
     pub fn mask(&self) -> Self {
         let mut conns = BTreeMap::new();
         for (k, v) in &self.conns {
-            conns.insert(k.to_string(), mask_string(v, 3));
+            let mut value = v.clone();
+            if k.to_lowercase() == "access_key_id" || k.to_lowercase() == "secret_access_key" {
```

Comment:
not only these 2 keys needs masking.
please refer to fn `parse_uri_location`.
if you are not formiliar with this field,  still masking all values is ok.

Reply:
ok, I will keep masking all values